go/logger/standard.go

```diff
@@ -439,6 +451,22 @@ func (log *Standard) processBuffer() {
 	}
 }
 
+func (log *Standard) logWithTimeout(handle uint64, xl ExternalLogger, e *entry) {
+	ch := make(chan struct{})
```

Comment:
This is OK for now, but I worry this might be expensive to do on every log call, across every window that may be open.